users/tracing/users.go

```diff
@@ -13,11 +13,10 @@ import (
 )
 
 const (
-	saveOp             = "save_op"
-	retrieveByEmailOp  = "retrieve_by_email"
-	generateResetToken = "generate_reset_token"
-	updatePassword     = "update_password"
-	sendPasswordReset  = "send_reset_password"
+	saveOp              = "save_op"
+	retrieveByEmailOp   = "retrieve_by_email"
```

Comment:
It should be `retrieve_by_email_op`?

Reply:
What is this `Op` and is it really needed?